templates/github/.github/workflows/ci.yml.j2

```diff
@@ -71,6 +71,11 @@ jobs:
         run: flake8 --config flake8.cfg
       {%- endif %}
 
+      {% if extra_linting -%}
+      - name: Run extra lint checks
+        run: [ -x .ci/scripts/extra_linting.sh ] && .ci/scripts/extra_linting.sh
+      {%- endif %}
```

Comment:
Sorry, forgot to mention: You need to remove the ifs and the docs for this flag again.
The whole PR would just be lines 75-76 (and the changelog).